.github/workflows/build.yml

```diff
@@ -23,7 +23,23 @@ jobs:
       matrix:
         arch: [amd64]
         version: [39, 40, rawhide]
-        flavor: [base, silverblue, kinoite]
+        flavor:
+          [
```

Comment:
why the use of multi-line bracketed list vs the hyphen-prefixed list syntax used elsewhere?

Reply:
That's what was here previously